dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/AnnotationBean.java

```diff
@@ -288,6 +296,73 @@ private Object refer(Reference reference, Class<?> referenceClass) { //method.ge
         return referenceConfig.get();
     }
 
+    /**
+     * refer 方法的公开版，将注解参数Reference换为 ReferenceConf 类
+     * @param reference
+     * @param referenceClass
+     * @return
+     */
+    public Object referNew(RefConf reference, Class<?> referenceClass) { //method.getParameterTypes()[0]
+        String interfaceName;
+
+        if (! "".equals( reference.interfaceName)) {

```

Comment:
Too much duplication of code with "com.alibaba.dubbo.config.spring.AnnotationBean#refer".

Reply:
yes , for convenience, i just copied the refer method to referNew. I  will fix it by extract a common method.

Reply:
@zhouzhipeng Existing refer methods is private, you can rename refer to refer0,  I think use refer better than referNew.  Of course, it's all depends on you, thank you very much for your contribution.